services/vision/obstaclesdepth/obstacles_depth.go

```diff
@@ -149,11 +152,14 @@ func (o *obsDepth) obsDepthWithIntrinsics(ctx context.Context, src camera.VideoS
 	if o.intrinsics == nil {
 		return nil, errors.New("tried to build obstacles depth with intrinsics but no instrinsics found")
 	}
-	pic, release, err := camera.ReadImage(ctx, src)
+	imgBytes, mimeType, err := src.Image(ctx, utils.MimeTypeRawDepth, nil)
```

Comment:
@bhaney similar thing here... should we be checking the context first for a mimetype? Also is raw depth the right mimetype to use here?